### DISCUSSION TITLE
Criação de Site Utilizando o Wix

### DISCUSSION THREAD

**Discussion by dsoldera** [frontendbr/forum#425] [Perguntas e Respostas]
Olá Pessoal...

Alguém já tentou criar um site bem simples usando o WIX e precisou fazer alguns ajustes???

Eu estou com um aqui e estou com uns bugs.... nao sei como poder arrumar?

Alguém tem alguma ideia ou experiência para compartilhar???

Exemplo: https://gazioli.wixsite.com/fotos

**Comment by joao-alberto** [comment]
Olá,

Eu acredito que o Wix não forneça acesso ao código fonte do site (posso estar errado), então fica um pouco difícil dar ajuda, eu recomendo entrar em contato com o [suporte](https://support.wix.com/pt/).

Mas se descrever mais detalhes de quais problemas esta enfrentando, talvez possamos ajudar.

😃

**Comment by rubenmarcus** [comment]
@dsoldera  https://br.wordpress.com/ Usar o free do wordpress não seria melhor? É uma opção no projeto?

**Comment by dsoldera** [comment]
Oi @rubenmarcus. È sim a melhor opção. Vou orientar meu cliente pra fazermos usando o Wordpress.

Grata.

**Comment by JoseCage** [comment]
AInda sobre o WordPress, @dsoldera se quiserem fazer coisas mais avançadas com o WordPress e também ter maior controle de um modo geral, recomendo fazer  (o cliente no caso) um investimento e usar logo o **WordPress.org** [wordpress.org](https://br.wordpress.org)

**Comment by tk21bs** [comment]
@dsoldera o ruim do wix é a burocracia de manutenção, pois nem FTP voce consegue acessar, logo, não consegue alterar arquivos, apenas layout, sou corretor de imóveis e recentemente tive que migrar o site de minha imobiliária para uma plataforma de código proprietário na Vipcom Sistemas, se precisar segue 
http://www.vipcomsistemas.com.br/ dependendo da área que voce quer criar o seu site, eles já tem o site pronto.

**Comment by AgenciaColors** [comment]
Olá @dsoldera existem diversas empresas no brasil especializadas na [criação de sites e otimização de sites](https://agenciacolors.com.br/solucoes/criacao-de-sites/), a Agência Colors é uma boa referência em atendimento e qualidade com seus clientes há mais de 5 anos no mercado. Caso deseje você também poderá optar por criadores gratuitos como Wix, Google Sites e outros!
